README.md

```diff
@@ -1,171 +1,261 @@
 
-This addon provides two decorators: `@localStorage` and `@sessionStorage`. 
-Both work identically, the only difference is the backing storage used. 
-`@localStorage` persists data in `window.localStorage` available across browser 
-sessions, while `@sessionStorage` persists data in `window.sessionStorage` only for 
-the duration of the current page session.
+### 2. Pre-instantiated Storage Instances
+
+For convenience, pre-instantiated TrackedStorage instances are provided:
+
+```js
+import { trackedLocalStorage, trackedSessionStorage } from 'ember-local-storage-decorator';
+import Component from '@glimmer/component';
+
+export default class MyComponent extends Component {
+  get currentUser() {
+    return trackedLocalStorage.getItem('user');
+  }
+
+  saveUser = (user) => {
+    trackedLocalStorage.setItem('user', user);
+  }
+}
+```
+
+These instances use the default prefix and are ready to use immediately.
+
+### 3. Property Decorators
+
+For a more traditional approach, use the `@localStorage` and `@sessionStorage` decorators to bind class properties directly to storage:
 
 ```js
 import { localStorage, sessionStorage } from 'ember-local-storage-decorator';
 import Component from '@glimmer/component';
 
 export default class MyComponent extends Component {
-  @localStorage foo;
-  @sessionStorage bar;
+  @localStorage user;
+  @sessionStorage tempData;
 }
 ```
 
-Decorate a class property with `@localStorage` or `@sessionStorage` to bind it 
-to the respective storage. It will attach a getter to read the value from storage 
-and a setter to write changes to storage.
+The decorators attach a getter to read the value from storage and a setter to write changes to storage.
+
+### Basic Usage
 
 ```js
 const Klass = class {
   @localStorage foo;
-  @sessionStorage bar;
 }
 const klass = new Klass();
 
 klass.foo = 'baz';
-window.localStorage.getItem('foo'); // '"baz"'
-
-klass.bar = 'qux';
-window.sessionStorage.getItem('bar'); // '"qux"'
+klass.foo; // 'baz'
 ```
 
-You may specify another key to be used in storage as an argument to the
-decorator.
+### Custom Storage Key
+
+You may specify a different key to be used in storage:
 
 ```js
 const Klass = class {
   @localStorage('bar') foo;
-  @sessionStorage('baz') qux;
 };
 const klass = new Klass();
 
-klass.foo = 'baz';
-window.localStorage.getItem('bar'); // '"baz"'
-
-klass.qux = 'quux';
-window.sessionStorage.getItem('baz'); // '"quux"'
+klass.foo = 'baz'; // stored under key 'bar'
 ```
 
-The value is stored as a JSON string in storage. Therefore only values
-which can be serialized to JSON are supported.
+### `@sessionStorage` Decorator
 
-Objects (and arrays) are deep frozen to avoid leaking state. Getter returns a
-frozen copy after setting a value.
+The `@sessionStorage` decorator works identically to `@localStorage` but uses `window.sessionStorage` instead:
 
 ```js
-window.localStorage.setItem('foo', [{ a: 'b' }]);
+const Klass = class {
+  @sessionStorage tempData;
+};
+```
```

Comment:
The readme does not contain a dedicated chapter for `@localStorage` decorator anymore. The basic usage and custom storage key chapters apply to both decorators. Even though it only uses `@localStorage` to illustrate. Did you maybe removed `@sessionStorage` from those examples by accident?